HeroLib/Events/Main.lua

```diff
@@ -419,6 +420,88 @@ HL:RegisterForEvent(function(Event, MessageType, Message)
   end
 end, "UI_ERROR_MESSAGE")
 
+--- ========================= UNIT UPDATE FUNCTIONS ============================
```

Comment:
Should localize Unit.Nameplate there, like wrap it into a do..end block and add a `local Nameplate = Unit.Nameplate`.
Considering how often the lookup is done, it speedup a little bit the thing.